torchaudio/sox_effects/sox_effects.py

```diff
@@ -1,4 +1,5 @@
-from typing import List, Tuple
+from typing import List, Tuple, Union
```

Comment:
@bhargavkathivarapu Can you remove unused `Union`?

Reply:
#1052